module/documents/chat-message.mjs

```diff
@@ -555,6 +551,30 @@ export default class ChatMessage5e extends ChatMessage {
     afterElement.insertAdjacentElement("beforebegin", enchantmentApplication);
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Display the effects tray with effects the user can apply.
+   * @param {HTMLLiElement} html  The chat card.
+   * @protected
+   */
+  _enrichUsageEffects(html) {
+    const item = this.getAssociatedItem();
+    let effects;
+    if ( this.getFlag("dnd5e", "type") === "usage" ) {
```

Comment:
Should this be `"type"` or should it be `"messageType"`? `"messageType"` is used elsewhere